### ISSUE TITLE
Rule Change: (fill in)

### ISSUE THREAD

**Issue by Dhaxor** [eslint/eslint#16280]
### What rule do you want to change?

function-paren-newline

### What change to do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
function sum(arg1,arg2) {
   return arg1 + arg2;
}
```

### What does the rule currently do for this code?

It just moves the function parentheses to a new line and not the arguments of a function declaration.

### What will the rule do after it's changed?

The rule would put the function declaration arguments on a new line.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by aladdin-add** [comment]
Thanks for the input. As [stated in the README](https://github.com/eslint/eslint#stylistic-rule-updates), we actually aren't making any further updates to rules that enforce stylistic preferences. If you have a strong preference for such a change, you can always copy the existing rule into a custom rule and make it behave the way you want.
